express/parsers/structure.py

```diff
@@ -1,9 +1,36 @@
 import io
+import os
 import pymatgen as mg
+from pymatgen.io.xyz import XYZ
 from ase.io import read, write
-
 from express.parsers import BaseParser
 from express.parsers.mixins.ionic import IonicDataMixin
+import rdkit
+from rdkit import Chem
+"""
+    openbabel & pybel work together to generate the molecule string that
+    can then be converted into an InChI string by rdkit.
+
+    openbabel & pybel can be imported when the butler-venv is running.
+    Without the butler-venv, openbabel is not installed and therefore
+    both imports will fail.
+
+    When the butler-venv is not installed, inchi generation will be 
+    set to 'None' and essentially skipped for the purpose of testing.
+
+    Without the 'try' statements ExPrESS will fail due to import errors.
+
+    If the openbabel import is successul, then pybel will be imported.
+    If the openbabel import fails, then get_inchi & get_inchi_key will
+    be set to 'None'
+"""
+try:
+    import openbabel
+    import pybel
+    inchi_run  = 1
+except ImportError:
+    print("WARNING: In Structure.py openbabel & pybel import failed. Inchi's will be turned off!")
+    inchi_run = 0
 
```

Comment:
I am not sure what the best option here is. We could leave as a try statement, or create another unction that tests these separately either within `structure.py` or within another python script within `ExPrESS`.Please let me know your thoughts.

We need to test whether openbabel and pybel can be installed. Because without this test the code will always fail in reviews because those imports only exist when butler is initiated.

Reply:
Add in log.level - express should already have a log file defined.